### PR TITLE
Fix Subscribe Changes Form -- Updated to Cred File

### LINKED ISSUE #335: Document users of CHIME
Add a quick form (like a little side pop-up) to ask people to tell us:
- who they are (e.g. hospital admin, hobbyist; their name if they want),
- where they are using it (e.g. MGH, LA county)

We currently have no insight into who is using the tool unless they explicitly email us or say so in the CfP slack.

### LINKED ISSUE #454: Subscribe to updates field
Add a field to subscribe to updates on the dashboard.
Include: email address, name, and affiliation

### LINKED ISSUE #516: ["bug"] 1.1.3.rc4 with Subscribe Changes Feature breaks on submission
### Description:
When submitting your information to subscribe to updates, the process fails on the google api package that calls the spreadsheet.
### Steps to reproduce:
1. Enter name, email and affliiation
2. Click Submit

### Expected behavior:
Information shows up on sheet.
### What I got instead:
(screenshots if applicable)
Screen Shot 2020-04-09 at 12 38 41

This is currently happening in preprod (https://penn-chime.bus.phl.io)

### LINKED ISSUE #548: ["bug"] Newsletter subscription is failing due to _private_key_pkcs8_pem_ error
Subscribe to newletter failed.
image

### PR DESCRIPTION
**Pull request by mangalap123**
### Link to issue
- Fixes #548 

### Changelog entry
- Moved the google API key data to json file
- Passed json file to  Google Service account credentials.

### Additional info
- Anything else reviewers and future readers should know

### CODE REVIEW

File: src/penn_chime/presentation.py [hunk 1]
```diff
@@ -433,9 +433,15 @@ def subscribe(st_obj):
     affiliation = st_obj.text_input (label="Enter Affiliation", value="", key="na_upper_2")
     if st_obj.button (label="Submit", key="ta_submit_1"):
         row = [email, name, affiliation]
-        send_subscription_to_google_sheet(st_obj, row)
+        send_subscription_to_google_sheet_secret_json(st_obj, row)
 
-def send_subscription_to_google_sheet(st_obj, row):
+def send_subscription_to_google_sheet_secret_json(st_obj, row):
+    json_secret = "/Users/mpatil763/opensource/COVED-19/repo/chime/src/penn_chime/client_secret.json"
```
Comment (MooseQuest) [inline src/penn_chime/presentation.py]: This is pointed at the wrong directory. Should be:

```suggestion
    json_secret = "/mnt/google-api-creds/client_secret.json"
```

**Review by MooseQuest — CHANGES_REQUESTED** [review]
Hi @mangalap123 -- see the one edit that I have below.

**Comment by jlubken** [comment]
These files moved:
1. presentation.py -> src/penn_chime/view/st_display.py
2. spreadsheet.py -> src/penn_chime/view/spreadsheet.py

File: src/penn_chime/presentation.py (hunk 1 above)
Comment (MooseQuest) [reply src/penn_chime/presentation.py]: This has been updated.

**Review by MooseQuest — APPROVED** [review]
The updates have been applied as requested. Approving and then merging.

File: src/penn_chime/presentation.py (hunk 1 above)
Comment (mangalap123) [reply src/penn_chime/presentation.py]: Reviewed the changes in st_display.py.

**Pull request merged**